components/proxy/conf/Caddyfile

```diff
@@ -235,10 +235,30 @@ https://{$GITPOD_DOMAIN} {
 			base_domain {$GITPOD_DOMAIN}
 		}
 
-		# note: no compression, as that breaks streaming for headless logs
+		forward_auth server.{$KUBE_NAMESPACE}.{$KUBE_DOMAIN}:3000 {
+			uri /feature-flags/slow-database
+			copy_headers X-Gitpod-Slow-Database
+		}
+
+		@slow {
+			header X-Gitpod-Slow-Database "true"
+		}
+
+		@fast {
```

Comment:
```suggestion
		@fast not @slow
```
Is a tad shorter.

Also, we could move those matchers one level up, so we can re-use them in other handlers (but I think that is a follow-up). :100:

Reply:
Unfortunately, negating a named matcher with `not` isn't possible: https://github.com/caddyserver/caddy/issues/4112. So I think we are stuck with the more verbose version.

I'll move the matchers in a followup PR when we re-use them in other `handle` blocks.